src/test/java/com/google/dicomwebfuse/dao/FuseDaoHelperTest.java

```diff
@@ -111,20 +99,6 @@ void testShouldReturnFiveThousandStudiesIfStudiesCountFiveThousand()
     assertEquals(expectedStudiesCount, actualStudyList.size());
   }
 
-  @Test
```

Comment:
Why remove these test cases?

Reply:
as i responded https://github.com/GoogleCloudPlatform/healthcare-api-dicom-fuse/pull/28/files/463d95ee32b65c83d4a621575be8e7f72278e855#r326368130 if we want to open unlisted files we no longer need logic with exception for large folder